recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set package = "dj-static" %}
```

Comment:
Glad we reached a consensus.

Though I did write up some guidelines and have tried to be a stickler. Some experimentation can be good and will help us learn what works. Maybe we should have some PEP like system for hardening the standard over time once we have had a chance to learn what helps and hurts recipe maintenance and readability. Totally up for discussion on those points and they are completely orthogonal to the acceptability of this proposal.